### PR TITLE
[FIF-78] Creates the base survey structures and creates the back-end for the survey summary results. 

### PR DESCRIPTION
**Pull request by DavidJGapCR**

### CODE REVIEW

File: edfi.fif.database/migrations/sqls/20200701164722-create-survey-result-views-up.sql [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+    surveyquestion.question
+  FROM 
+    fif.survey
+      INNER JOIN 
+        fif.surveyquestion 
+          ON survey.surveykey = surveyquestion.surveykey;
+
+---
+
+CREATE OR REPLACE VIEW fif.SurveySummaryAnsweres
```
Comment (jleiva-gap) [inline edfi.fif.database/migrations/sqls/20200701164722-create-survey-result-views-up.sql:55]: Name should be **SurveySummaryAnswers**

File: edfi.fif.api/src/graphql/entities/survey/surveysummaryansweres.entity.ts [hunk 2]
```diff
@@ -0,0 +1,28 @@
+import { ViewEntity, ViewColumn, PrimaryColumn } from 'typeorm';
+
+@ViewEntity({ schema: 'fif', name: 'surveysummaryansweres', synchronize: false })
+export default class SurveySummaryAnsweresEntity {
```
Comment (jleiva-gap) [inline edfi.fif.api/src/graphql/entities/survey/surveysummaryansweres.entity.ts:4]: Name should be **SurveySummaryAnswersEntity**

File: edfi.fif.database/migrations/sqls/20200701164722-create-survey-result-views-up.sql (hunk 1 above)
Comment (DavidJGapCR) [reply edfi.fif.database/migrations/sqls/20200701164722-create-survey-result-views-up.sql]: Fixed

File: edfi.fif.api/src/graphql/entities/survey/surveysummaryansweres.entity.ts (hunk 2 above)
Comment (DavidJGapCR) [reply edfi.fif.api/src/graphql/entities/survey/surveysummaryansweres.entity.ts]: Fixed.

**Pull request merged**